easybuild/easyconfigs/i/icc/icc-2015.2.164-GCC-4.9.2.eb

```diff
@@ -0,0 +1,25 @@
+name = 'icc'
+version = '2015.2.164'
+
+homepage = 'http://software.intel.com/en-us/intel-compilers/'
+description = "C and C++ compiler from Intel"
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['l_ccompxe_%(version)s.tgz']
+
+gcc = 'GCC'
+gccver = '4.9.2'
+versionsuffix = '-%s-%s' % (gcc, gccver)
+
+dependencies = [(gcc, gccver)]
+
+dontcreateinstalldir = 'True'
+
+# license file
+#import os
+#license_file = os.path.join(os.getenv('HOME'), "licenses", "intel", "license.lic")
+license_file = '/import/bc2/soft/source/i/icc/COM_L__CPP_RSCN-FRFDSRTL.lic'
```

Comment:
switch back to the commented out line

we now know where you store your license file! mwoehahahaha! :P